### PR TITLE
fix bug in search.

### PR DESCRIPTION
**Pull request by jonhartzler**

### CODE REVIEW

**Comment by jgondron** [comment]
👍

File: src/layout/SearchBox.jsx [hunk 1]
```diff
@@ -67,7 +67,7 @@ var SearchBox = React.createClass({
 
   setSearchTerm(searchTerm) {
     this.setTerm(searchTerm);
-    
+
```
Comment (rfox2) [inline src/layout/SearchBox.jsx]: OK, so why remove the whitespace? Are you prejudiced against it?

File: src/layout/SearchBox.jsx [hunk 2]
```diff
@@ -84,8 +84,7 @@ var SearchBox = React.createClass({
   },
 
   setTerm: function(term) {
-    var cleanTerm = encodeURIComponent(term);
-    this.setState({searchTerm: cleanTerm});
+    this.setState({searchTerm: term});
```
Comment (rfox2) [inline src/layout/SearchBox.jsx]: This looks too clean.

**Comment by rfox2** [comment]
👎

**Pull request merged**